### ISSUE TITLE
Can't find emacs in your PATH

### ISSUE THREAD

**Issue by AllenDang** [jimeh/emacs-builds#12]
Build 6-26 and 6-27 failed to start.

Run `doom sync` reports can't find emacs in your path.

Run `emacs` from terminal reports
```
Error using execdir /Applications/Emacs.app/Contents/MacOS/:
emacs: dlopen(/Applications/Emacs.app/Contents/MacOS/../native-lisp/28.0.50-b01b8d59/preloaded/window-0d1b8b93-63c191c3.eln, 1): image not found
```

**Comment by darrenkenny** [comment]
I'm seeing the same, but not using doom, and it is VERY subtle, but in the distributed builds the version in the path uses `-` instead of `.`, i.e. the path that exists is `28-0-50-b01b8d59`, but `dlopen` is looking for the path `28.0.50-b01b8d59`.

**Comment by jimeh** [comment]
Sorry for the late replay, this issue should be fixed in yesterday's nightly build: [Emacs.2021-06-30.8ebd8e1.master](https://github.com/jimeh/emacs-builds/releases/tag/Emacs.2021-06-30.8ebd8e1.master)

There's some more changes coming in this area in the next nightly build that'll show up in a couple of hours thanks to recent fixes in the Emac's master branch for [bug#49270](https://debbugs.gnu.org/cgi/bugreport.cgi?bug=49270) and [bug#49271](https://debbugs.gnu.org/cgi/bugreport.cgi?bug=49271). Changes needed in my build setup for them are already in place, and have worked locally for me, so it should be fine :D

However, this yet one more reason why I need to get some basic tests into the build process which actually tries to at the very least execute some elisp via the build.

**Comment by jimeh** [comment]
I'm pretty sure this is resolved, as of the 2012-06-30 nightly, affected builds are 2021-06-26 to 2021-06-29.

So I'm closing this issue, and hopefully there'll be no need to re-open :)
